low-level/framework/base/sspl_constants.py

```diff
@@ -61,19 +61,23 @@
 # required only for init
 component = 'sspl/config'
 file_store_config_path = '/etc/sspl.conf'
+roles = ['vm', 'cortx', 'ssu', 'gw', 'cmu']
 salt_provisioner_pillar_sls = 'sspl'
 salt_uniq_attr_per_node = ['cluster_id']
 salt_uniq_passwd_per_node = ['RABBITMQINGRESSPROCESSOR', 'RABBITMQEGRESSPROCESSOR', 'LOGGINGPROCESSOR']
+# This file will be created when sspl is being configured for node replacement case
+REPLACEMENT_NODE_ENV_VAR_FILE = '/etc/profile.d/set_replacement_env.sh'
 
 # Initialize to default values
 node_key_id = 'srvnode-1'
 CONSUL_HOST = '127.0.0.1'
 CONSUL_PORT = '8500'
+CONSUL_PATH = '/usr/bin/'
 
 # TODO Keep only constants in this file.
 # other values(configs) should come from cofig.
 # Check if SSPL is configured
-if os.path.exists(SSPL_CONFIGURED):
+if os.path.exists(SSPL_CONFIGURED) or True:
```

Comment:
in shell script as we used to call "conf_based_sensor_enabled.py" after the SSPL_CONFIGURED file is touched, now since I am importing the method from "conf_based_sensor_enabled.py" it checks the imports of storage_type at the start of sspl_conf.py and at this time SSPL_CONFIGURED is no touched, so what could be done?

Reply:
for testing I used "or True"

Reply:
touch SSPL_CONFIGURED then before importing from conf_based_sensor_enabled
Make sure to remove True from here